src/routes/RoutesWrapper.ts

```diff
@@ -183,14 +183,32 @@ class RoutesWrapper {
         ],
 
         ['/items', 'items', 'get', null, itemInstance, null, false],
+        ['/items/disabled', 'items', 'get', null, itemInstance, null, false],
         ['/items/{_id}', 'items', 'get', generateIDParam(), itemInstance, null, false],
         ['/items/{_id}', 'items', 'put', generateIDParam(), itemInstance, itemWithoutId, false],
-        ['/items/{_id}', 'items', 'delete', generateIDParam(), null, null, false],
+        [
+            '/items/{_id}',
+            'items',
+            'patch',
+            [...generateIDParam(), ...generateQueryParam(1, [{ name: 'availability', type: 'boolean' }])],
+            null,
+            null,
+            false,
```

Comment:
Remove trailing comma.
```suggestion
            false
```